src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -173,6 +173,14 @@ parameters:
     items:
       type: string
     collectionFormat: csv
+  nodeDirectAccessExpiryTimeParam:
+    name: expiryTime
+    in: query
```

Comment:
If we're providing parameters for a POST endpoint it should be in the body of the POST, not a query string parameter.

Reply:
Change done